### PR TITLE
Check-in for the addition of the SpatialPersistence system

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

Check-in for the addition of the SpatialPersistence system

## Related Submodule Changes

- https://github.com/XRTK/SpatialPersistence/pull/9
- https://github.com/XRTK/SpatialPersistence.ASA/pull/6

### CODE REVIEW

**Comment by StephenHodgson** [comment]
Builds not passing

**Comment by StephenHodgson** [comment]
builds are failing

**Comment by SimonDarksideJ** [comment]
What the Frell Unity/MS  Looks like "something" for updated in the WinMR package.  I'll look in to it

File: XRTK-Core/Packages/manifest.json [hunk 1]
```diff
@@ -24,14 +24,14 @@
     "com.unity.ide.vscode": "1.2.4",
     "com.unity.mobile.android-logcat": "1.2.3",
     "com.unity.test-framework": "1.1.29",
+    "com.unity.xr.management": "4.0.7",
```
Comment (StephenHodgson) [inline XRTK-Core/Packages/manifest.json:27]: Please reference this in the package that requires it

File: XRTK-Core/Assets/XRTK.Generated/SymbolicLinkSettings.asset [hunk 2]
```diff
@@ -34,3 +34,6 @@ MonoBehaviour:
   - sourceRelativePath: /Submodules/Ultraleap/XRTK.Ultraleap/Packages/com.xrtk.ultraleap/Profiles~
     targetRelativePath: /XRTK-Core/Assets/XRTK.Generated/Ultraleap/Profiles
     isActive: 1
+  - sourceRelativePath: /Submodules/glTF/XRTK.glTF/Packages/com.xrtk.gltf
+    targetRelativePath: /XRTK-Core/Packages/com.xrtk.gltf
+    isActive: 1
```
Comment (StephenHodgson) [inline XRTK-Core/Assets/XRTK.Generated/SymbolicLinkSettings.asset:39]: please revert this change

**Review by StephenHodgson — APPROVED** [review]
